app/controllers/registration_controller.rb

```diff
@@ -0,0 +1,30 @@
+# RegistrationController
+#
+# Handles incoming registration requests from tool
+# consumers.
+class RegistrationController < ApplicationController
+  include RegistrationHelper
+
+  def register
+    # Fetch the a custom tool consumer profile
+    tcp_helper = Registration::ToolConsumerProfile.new(URI.parse(params[:tc_profile_url]), jwt_access_token)
```

Comment:
I agree. Originally I had this in the helper but it felt a little odd having multiple classes in that file.